pkg/handlers/scripts/install.sh

```diff
@@ -10,6 +10,44 @@ function fail {
 	echo "Error: $msg" 1>&2
 	exit 1
 }
+function prompts_can_prompt() {
+	# Need the TTY to accept input and stdout to display
+	# Prompts when running the script through the terminal but not as a subshell
+	if [ -t 1 ] && [ -c /dev/tty ]; then
+		return 0
+	fi
+	return 1
+}
+function read_installation_path {
+	if ! prompts_can_prompt ; then
+		echo "Automatically setting default path: ${OUT_DIR}, shell is not interactive"
+		return
+	fi
+	printf "Select installation path:[press 'Enter' to use default path ${OUT_DIR}]: \n"
+	filepath=
+	while true ; do
+		read -r -p "Path:  " filepath </dev/tty
```

Comment:
The examples from the PR comment look like this prints "Path:   " to the screen. It seems noisy. They just entered the path; I don't think we need to print it out again.